packages/adblocker-electron-preload/src/index.ts

```diff
@@ -7,24 +7,28 @@
  */
 
 import { ipcRenderer } from 'electron';
+import { DOMMonitor, IBackgroundCallback } from '@cliqz/adblocker-content';
 
-import {
-  DOMMonitor,
-  IBackgroundCallback,
-  IMessageFromBackground,
-  injectScript,
-} from '@cliqz/adblocker-content';
+type CosmeticFiltersResponse = {
+  active: boolean;
+  error?: string;
+};
```

Comment:
```suggestion
interface ICosmeticFiltersResponse = {
  active: boolean;
  error?: string;
};
```
To match the naming convention of `IBackgroundCallback`